core/build_playerglobal/src/lib.rs

```diff
@@ -0,0 +1,103 @@
+//! An internal Ruffle utility to build our playerglobal
+//! `library.swf`
+
+use std::collections::HashSet;
+use std::fs::File;
+use std::path::PathBuf;
+use std::process::Command;
+use swf::DoAbc;
+use swf::Header;
+use swf::SwfStr;
+use swf::Tag;
+use walkdir::WalkDir;
+
+/// If successful, returns a list of paths that were used. If this is run
+/// from a build script, these paths should be printed with
+/// cargo:rerun-if-changed
+pub fn build_playerglobal(repo_root: PathBuf) -> Result<Vec<PathBuf>, Box<dyn std::error::Error>> {
+    let mut classes_dir = repo_root.clone();
+    classes_dir.push("core/src/avm2/playerglobal/classes/");
+
+    let mut asc_path = repo_root.clone();
+    asc_path.push("core/build_playerglobal/asc.jar");
+
+    let mut out_path = repo_root.clone();
```

Comment:
If we are to not checkout `library.swf` into git, this can be put in Rust's output directory.
See https://stackoverflow.com/questions/47217283/how-do-i-generate-a-text-file-during-compile-time-and-include-its-content-in-the